contrib/k8s/charts/open-service-broker-azure/README.md

```diff
@@ -57,11 +57,12 @@ $ export AZURE_CLIENT_SECRET=<password>
 ## Installing the Chart
 
 Installation of this chart is simple. First, ensure that you've [added the
-`azure` repository](../README.md#installing-charts). Then, install from the
+`azure` repository]. Then, install from the
```

Comment:
This doesn't look right. This doesn't render as a link.

Reply:
Yep! sorry, forgot to delete the [ ]